mmv1/templates/terraform/examples/cloudrunv2_service_directvpc.tf.erb

```diff
@@ -1,7 +1,7 @@
 resource "google_cloud_run_v2_service" "<%= ctx[:primary_resource_id] %>" {
   name     = "<%= ctx[:vars]['cloud_run_service_name'] %>"
```

Comment:
Double-checking that this name is valid (as opposed to just `cloudrun-service`).

Reply:
These have associated config blocks as described at https://googlecloudplatform.github.io/magic-modules/develop/test/test/, you can see the rendered value in https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-10524-old..auto-pr-10524